pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -912,6 +916,14 @@ public boolean enableSegmentRelocatorLocalTierMigration() {
     return getProperty(ControllerPeriodicTasksConf.SEGMENT_RELOCATOR_ENABLE_LOCAL_TIER_MIGRATION, false);
   }
 
+  public long getSegmentRelocatorExternalViewCheckIntervalMs() {
+    return getProperty(ControllerPeriodicTasksConf.SEGMENT_RELOCATOR_EXTERNAL_VIEW_CHECK_INTERVAL_MS, 1000);
```

Comment:
Should we use the constant introduced in `RebalanceConfigConstants` as default?

Reply:
yeah, can do. I usually just avoid deps for simple stuff like this.